logger/README.md

```diff
@@ -70,3 +71,28 @@ lumberjackConfig:
   # Whether to compress the log file, the compression method is gzip
   compress: false
 ```
+
+#### coustom logger
+
+Custom logger needs to implement the logger interface in the logger package
+
+```go
+type Logger interface {
+	Info(args ...interface{})
+	Warn(args ...interface{})
+	Error(args ...interface{})
+	Debug(args ...interface{})
+
+	Infof(fmt string, args ...interface{})
+	Warnf(fmt string, args ...interface{})
+	Errorf(fmt string, args ...interface{})
+	Debugf(fmt string, args ...interface{})
```

Comment:
pls add Fatalf()

Reply:
This pr has added Fatal() &  Fatalf() https://github.com/apache/dubbo-go/pull/1482,  Wait for this pr to be merged and then merge this pr.